code/src/styledComponents/styled_components.js

```diff
@@ -0,0 +1,124 @@
+import styled from 'styled-components';
+
+// Header
+export const Title = styled.h1`
+  text-align: center;
+  font-size: 36px;
+  color: white;
+  padding-top: 30px;
+  font-family: 'Nunito', sans-serif;
+  font-weight: 700;
+    @media (min-width: 768px) {
+      font-size: 66px;
+    }
+`
```

Comment:
Consistency with semi-colons: If you are using them, I believe you should have a semi-colon separating every styled-component:

```
export const Title = styled.h1`
  text-align: center;
  font-size: 36px;
  color: white;
  padding-top: 30px;
  font-family: 'Nunito', sans-serif;
  font-weight: 700;
    @media (min-width: 768px) {
      font-size: 66px;
    }
`;
```